optaweb-vehicle-routing-docs/src/main/asciidoc/quickstart.adoc

```diff
@@ -0,0 +1,172 @@
++
+image::download.png[align="center"]
++
+. Extract the downloaded distribution ZIP file. The archive contains source files and a binary build of OptaWeb Vehicle Routing as well as the OptaWeb Vehicle Routing documentation.
++
+image::distribution.png[align="center"]
+
+=== Run OptaWeb Vehicle Routing
+
+After you download OptaWeb Vehicle Routing, use the `run.sh` script to run it.
+
+.Prerequisites
+* Internet access is available.
+When OptaWeb Vehicle Routing runs it uses third-party public services such as link:https://www.openstreetmap.org/about[OpenStreetMap] to display map tiles and provide search results.
+* Java 8 or higher is installed.
+* A directory called `openstreetmap` with at least one OSM file exists.
+* A country code to use in search queries is identified.
+
+.Procedure
+Enter the following command, where `<DATA_DIR>` is the parent of the `openstreetmap` directory, `<COUNTRY>` is the country that you downloaded data for and `<CODE>` is the code for that country:
+
+[source,bash]
+----
+./bin/run.sh <DATA_DIR> <COUNTRY>-latest.osm.pbf <CODE>
+----
+
+For example, if you want to run OptaWeb Vehicle Routing with Belgium and you downloaded the `belgium-latest.osm.pbf` from the Geofabrik website and saved it as `~/.vrp/openstreetmap/belgium-latest.osm.pbf`, enter the following command:
+
+[source,bash]
+----
+./bin/run.sh ~/.vrp/ belgium-latest.osm.pbf BE
+----
+
+=== Using air distance mode
+
+OptaWeb Vehicle Routing can work in air distance mode that calculates travel times based on the distance between two coordinates.
+Use this mode in situations where you need to get OptaWeb Vehicle Routing up and running as quickly as possible and do not want to use an OSM (OpenStreetMap) file.
+Air distance mode is only useful if you need to smoke-test OptaWeb Vehicle Routing and you do not need accurate travel times.
+
+.Prerequisites
+* Java 8 or higher is installed.
+* OptaWeb Vehicle Routing distribution archive is downloaded and extracted.
+
+.Procedure
+. Enter the `run.sh` script without arguments to start OptaWeb Vehicle Routing in air distance mode:
++
+[source,bash]
+----
+./bin/run.sh
+----
+
+. When prompted,  confirm that you want to start OptaWeb Vehicle Routing in air distance mode.
+
+=== Building OptaWeb Vehicle Routing
+
+If you do not have an executable distribution of OptaWeb Vehicle Routing,
+for example you have just cloned the Git repository from https://github.com/kiegroup/optaweb-vehicle-routing,
+you must build OptaWeb Vehicle Routing before you can run it.
+
+.Prerequisites
+* You have cloned the OptaWeb Git repository.
+
+.Procedure
+. To build OptaWeb Vehicle Routing, change directory to the root of the project with the `mvnw` script file:
++
+[source,bash]
+----
+cd optaweb-vehicle-routing
+----
+
+. Build with Maven:
++
+[source,bash]
+----
+./mvnw clean install -DskipTests
+----
+
+For more information about building OptaWeb Vehicle Routing, see the Development chapter.
```

Comment:
I assume that the development chapter will be provided later?

Reply:
Yes.